common/djangoapps/startup_configurations/validate_config.py

```diff
@@ -9,34 +9,17 @@ def validate_lms_config(settings):
     """
     validate_common_config(settings)
 
-    # validate feature based configurations
-    validate_marketing_site_config(settings)
-
 
 def validate_cms_config(settings):
     """
     Validates configurations for lms and raise ValueError if not valid
     """
     validate_common_config(settings)
 
-    # validate feature based configurations
-    validate_marketing_site_config(settings)
-
 
 def validate_common_config(settings):
     """
     Validates configurations common for all apps
     """
     if not getattr(settings, 'LMS_ROOT_URL', None):
         raise ValueError("'LMS_ROOT_URL' is not defined.")
-
-
-def validate_marketing_site_config(settings):
```

Comment:
> Why not call get_run_marketing_url every time and let the function check the cache? Forcing the caller to pass in a cached value doesn't make sense to me.

I have made response to your this comment but i am not sure why its not appearing in this view. If you go to the file `common/djangoapps/util/course.py` in this PR; you can see my comment there.